file-formats/ntty/type/zif_aff_ntty_v1.intf.abap

```diff
@@ -0,0 +1,70 @@
+INTERFACE zif_aff_ntty_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! $values { @link zif_aff_ntty_v1.data:co_profile }
+    "! $default { @link zif_aff_ntty_v1.data:co_profile.text }
+    ty_profile TYPE c LENGTH 20,
+
+    "! $minimum 0
+    "! $maximum 2147483647
+    ty_max_length TYPE i,
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">Language Dependent</p>
+      "! Language dependent
+      "! $showAlways
+      language_dependency TYPE abap_bool,
+      "! <p class="shorttext">Multiple Notes</p>
+      "! Multiple notes possible
+      "! $showAlways
+      multiple            TYPE abap_bool,
+      "! <p class="shorttext">Hide Title</p>
+      "! Hides the title
+      no_subject          TYPE abap_bool,
+    END OF ty_general_information,
+
+    "! <p class="shorttext">Content Settings</p>
+    "! Content settings
+    BEGIN OF ty_content_settings,
+      "! <p class="shorttext">Maximum Length</p>
+      "! Maximum length
+      max_length TYPE ty_max_length,
+      "! <p class="shorttext">Formatting Profile</p>
+      "! Formatting profile
+      profile    TYPE ty_profile,
+    END OF ty_content_settings,
+
+    "! <p class="shorttext">Notetype</p>
+    "! Notetype
+    BEGIN OF ty_main,
+      "! <p class="shorttext">Format Version</p>
+      "! Format version
+      "! $required
+      format_version      TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header              TYPE zif_aff_types_v1=>ty_header_60,
+      "! <p class="shorttext">General Information</p>
+      "! General information
+      general_information TYPE ty_general_information,
```

Comment:
> Would it be better to have the three bools on this level without an extra structure?

I think we came to the conclusion that it is easier to handle if there is a dedicated structure. If we have a specific/meaningful name for the section, it would be the best. For others, I think `general_information` is fine.

> Maybe, @schneidermic0, we should think about a general suggestion.

I agree

Reply:
We've already looked for a good name but couldn't find one, so we checked other objects and found the term "general_information". 
It sounds like that's ok, so I'd leave it as it is.

Reply:
We talked in our team about this issue. We recommend using `general_information` as you did. So this is fine for us.